src/ReleaseHistory.md

```diff
@@ -4,6 +4,7 @@
 
 * FEATURE: Add `max-file-size-in-kb` argument that allows filtering scan targets by file size. [#2494](https://github.com/microsoft/sarif-sdk/pull/2494)
 * BUGFIX: Fix false positive for `SARIF1002.UrisMustBeValid` for file URIs that omit the `authority`. [#2501](https://github.com/microsoft/sarif-sdk/pull/2501)
+* BREAKING: `absoluteAddress` property of `address` changed from `int`(32bit) to `BigInteger`(unlimited) to fix `Newtonsoft.Json.JsonReaderException: JSON integer XXXXX is too large or small for an Int32.` [#2502](https://github.com/microsoft/sarif-sdk/pull/2502)
```

Comment:
This PR follows the same fix as the one we did before for Location.Id.

Also listed the other int fields in the same `Address` class (not fixed in current PR), in the `description.txt` of the PR.

Let me know if we want this PR to fix the reported issue and publish a version then a separated PR to fix other fields we choose to. If this PR is reviewed all other properties can follow the same pattern.

Reply:
How about:
```Fix `Newtonsoft.Json.JsonReaderException: JSON integer XXXXX is too large or small for an Int32` by changing the `absoluteAddress` property of `address` from `int`(32bit) to `BigInteger`(unlimited).```

Reply:
agree, but now I notice that both of them may confuse the people looking at this checkin history fixed all integer in the solution,

How about something like:
BREAKING: Fix `Newtonsoft.Json.JsonReaderException: JSON integer XXXXX is too large or small for an Int32` 
for `absoluteAddress` property of `address` 
by changing it from `int`(32bit) to `BigInteger`(unlimited).

Reply:
Agreed.

Reply:
updated as discussed.